### ISSUE TITLE
Crash in MacOSX 10.13.6 while Soundforger was in backgrounds

### ISSUE THREAD

**Issue by RokuKent** [dvorka/mindforger#829]
Sorry can't give you the exact steps to repro. I suspect this is actually a Qt issue, might be fixed with a newer version. Or then this is due to a child no longer valid so a delete is done on an already deallocated one. 
[mindforge-crash-181002.txt](https://github.com/dvorka/mindforger/files/2439655/mindforge-crash-181002.txt)
 Stack trace enclosed as txt file.

**Comment by dvorka** [comment]
@RokuKent thank you for the issue and backtrace! Just to make sure that I understand it correctly: MindForger did't crash on start, but you were using it for some time and then it crashed, right?

**Comment by RokuKent** [comment]
Yes, I think it's running in background mode or switching to background mode as it does not crash during launch or using the app.
